charts/keycloakx/values.yaml

```diff
@@ -403,7 +403,7 @@ serviceMonitor:
   # Timeout for scraping
   scrapeTimeout: 10s
   # The path at which metrics are served
-  path: /auth/metrics
+  path: '{{ trimSuffix "/" .Value.http.relativePath}}/metrics'
```

Comment:
Please also update https://github.com/codecentric/helm-charts/blob/d14250633937796653405b12c5699bf6362b7d18/charts/keycloakx/templates/servicemonitor.yaml#L34 to render this string as a template.

Reply:
Fixed